.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: python
+python:
+  - "2.6"
+  - "2.7"
+
+
+install:
+  - pip install -r requirements.txt
+# command to run tests
+script: nosetests
```

Comment:
The `script` here means you run test with this script, so you can write a `test.py` and update `script` to something like `python test.py`. Then, let travis build successfully.